### PR TITLE
feat: help utilities

### PR DESCRIPTION
**Pull request by mdonnalley**
Extract help utilities from plugin-help into a separate library

@W-10293861@

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @heroku/cli
```
Comment (RodEsp) [inline .github/CODEOWNERS:1]: Is this right?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Salesforce.com
```
Comment (RodEsp) [inline LICENSE:3]: 2021?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    "chalk": "^4.1.2",
+    "indent-string": "^4.0.0",
+    "lodash": "^4.17.21",
+    "string-width": "^4.2.0",
+    "strip-ansi": "^6.0.0",
+    "widest-line": "^3.1.0",
+    "wrap-ansi": "^6.2.0"
+  },
+  "devDependencies": {
+    "@oclif/command": "^1.8.13",
```
Comment (RodEsp) [inline package.json:19]: This will still cause a circular dependency. Could we refactor the tests to not use it?

Comment (peternhale) [reply package.json:19]: This is part of the risk in our current dev practices. This form will not cause issues with customers during install, but may affect devs if installing for dev work on the module.

**Review by peternhale — APPROVED** [review]

**Review by RodEsp — APPROVED** [review]

**Pull request merged**